lib/pbench/server/indexing_tarballs.py

```diff
@@ -50,7 +50,7 @@ def _count_lines(fname):
 class Index:
     """class used to collect tarballs and index them"""
 
-    def __init__(self, name, options, idxctx, incoming):
+    def __init__(self, name, options, idxctx, incoming, archive, qdir):
```

Comment:
Can you isolate these changes to add the `archive` and `qdir` parameters in a separate commit from the signal handling changes?

Reply:
Sure and will work on the reviiew

Reply:
Also, add SIGHUP to the pbench-index `main` docstring with the other exceptions!

Reply:
ok